src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,35 @@
+package seedu.addressbook.commands;
+
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import java.util.List;
+
+/**
+ *
+ * Sorts the persons in Addressbook lexicographically based on the names of the persons.
+ * Displays the sorted list after sorting is done.
+ */
+
+public class SortCommand extends Command {
+    public static final String COMMAND_WORD = "sort";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Sorts and displays all persons in the address book according to lexicographical order of their names.\n"
+            + "Example: " + COMMAND_WORD;
+
+    private static final String MESSAGE_SUCCESS = "Successfully sorted the list.\n";
+
+
```

Comment:
Use blank lines sparingly. It's good to have blank lines to separate logical code blocks, but too many blank lines will increase the height of the code, which increases the amount of scrolling.